web/massastation/src/custom/api/usePost.ts

```diff
@@ -6,15 +6,15 @@ import { UseMutationResult, useMutation } from '@tanstack/react-query';
 
 // LOCALS
 
-export function usePost<T, P>(
+export function usePost<T>(
```

Comment:
Please, open a new MR for the rest of them. useDelete, usePut, etc... All should follow the same. No need in this MR but we need update the others too.

BTW, in the wallet repo too!

Reply:
done